src/ConfUtils.hs

```diff
@@ -0,0 +1,17 @@
+{-
+-- EPITECH PROJECT, 2024
+-- MyPandoc
+-- File description:
+-- ConfUtils
+-}
+
+module ConfUtils (getFomat) where
+
+import Conf (Conf(..), DocumentFormat(..))
+
+getFomat :: Conf -> String -> Conf
```

Comment:
Typo, should be `getFormat`, should rename everywhere.

Reply:
done